modules/Wordlist/README.md

```diff
@@ -10,17 +10,21 @@ thus in CLDF only a [`FormTable`](../../components/forms) is required.
 
 ## Introduction
 
-A proper handling of lexical data is one of the core aspects of CLDF. By now, many examples on how to handle certain kinds of data have been presented in talks but also implemented in different tools, including [LingPy](http://lingpy.org) and [EDICTOR](http://edictor.digling.org). Lexical data is to a large extent much more amenable to strict cross-linguistic formalization than other types of linguistic data, but it is obvious that not all language-specific problems can be handled at once. A general idea behind CLDF is to not be intimidated by apparent complexity but instead to try to amplify the power of standards step by step.
+Supporting proper handling of lexical data is one of the core goals of CLDF. By now, many examples on how to handle certain kinds of data have been presented in talks but also implemented in different tools, including [LingPy](http://lingpy.org) and [EDICTOR](http://edictor.digling.org). Lexical data is to a large extent much more amenable to strict cross-linguistic formalization than other types of linguistic data, but it is obvious that not all language-specific problems can be handled at once. A general idea behind CLDF is to not be intimidated by apparent complexity but instead to try to amplify the power of standards step by step.
 
-We orient our framework in `cldf` by basing lexical specifications as they can be added to word list data on a rough model of the linguistic sign (which can be a word but also a morpheme) which consists of a **language** in which the sign is expressed, a **meaning** that the sign expresses, and a **form** that denotates the meaning. Language specification is handled withing Glottolog, but specifications of linguistic varieties can define one specific variety within one given language, as long as Glottolog does not include it. Meaning specifications are handled within the Concepticon but can be left unlinked as long as the Concepticon offers the degree of fine-grainedness or the semantic field of interest. The latter will be the case in very specific dialect datasets where many cultural terms are of interest when studying language variation. In principle, however, both Glottolog and Concepticon offer ways to add new language varieties or concepts by providing modifications of the respective code base. 
+The framework for lexical specifications, as they can be added to word list data, is roughly oriented on the model of the linguistic sign (which can be a word but also a morpheme) which consists of a **language** in which the sign is expressed, a **meaning** that the sign expresses, and a **form** that denotates the meaning. 
+- Language specification should be done by referencing language catalogs like Glottolog, but specifications of linguistic varieties may need to be more fine-grained than e.g. Glottolog allows at the moment. 
```

Comment:
I suggest to reword:

> Language specification should be done by linking to reference catalogs like Glottolog, but specifications of linguistic varieties may need to be more fine-grained than e.g. Glottolog allows at the moment.

In this way, the term "reference catalog" is used instead of "language catalog".